packages/hooks/src/Provider.tsx

```diff
@@ -0,0 +1,94 @@
+import { JsonRpcSigner } from '@ethersproject/providers/src.ts/json-rpc-provider';
+import React from 'react';
+import { ethers } from 'ethers';
+import WalletConnectProvider from '@walletconnect/web3-provider';
+import Web3Modal from 'web3modal';
+
+export interface Web3ContextType {
```

Comment:
> Web3ContextType

But it's not a type, it's an interface! 😆

Reply:
classic